src/ui/common/src/components/notifications/NotificationListItem.tsx

```diff
@@ -97,25 +94,14 @@ export const NotificationListItem: React.FC<Props> = ({
           icon={faXmark}
           style={{
             cursor: 'pointer',
-            color: 'gray.600',
+            color: 'gray600',
```

Comment:
The MUI theme uses `gray.600` as the style. If you look at the X icon in the screenshot you provided, the styling isn't being reflected currently.

Reply:
Yes, it requires a period to separate the color and it's weight.
so ```gray.600``` would be the correct way to do this.

In the future, we should add some constants to have a safer way to code these. Missing a period is pretty easy to do and not something we can detect easily via our tools, so some consts will help. I'll file a linear task for this.